subprojects/gradle/runner/service/src/main/kotlin/com/avito/runner/service/worker/device/adb/listener/AdbDeviceEventsLogger.kt

```diff
@@ -6,67 +6,75 @@ import java.nio.file.Path
 
 internal class AdbDeviceEventsLogger(private val logger: Logger) : AdbDeviceEventsListener {
 
```

Comment:
The more I see massive changes in a huge interface the more I think about expressing it like a command pattern:

Event:
- command + arguments
- result
- duration